source/index.html.md

```diff
@@ -63,33 +56,45 @@ Any API method calls made from a non-whitelisted IP Address will be blocked.
 
 ```shell
 curl \
--H "Content-Type: application/json" \
```

Comment:
removed this because it seems like we were adding headers in sequence of them being mentioned so its in the section below